src/panels/meeting-times.js

```diff
@@ -1,45 +1,64 @@
 var React = require('react');
+var $ = require('jquery');
 var classNames = require('classnames');
+var time = require('../utils/time.js');
+var moment = require('moment')
 
-var meetingTimes = require('../data/meeting-times.json');
+var secrets = require('../secrets.js');
+var groupsURL = secrets.grootServicesURL + '/groups/sigs'
 
 var ROWS_PER_PAGE = 9;
+var REFRESH_TIMES_MS = 60 * 1000;
+var SWITCH_PAGE_MS = 10 * 1000;
 
 /**
  * Meeting times panel.
  */
 var MeetingTimesPanel = React.createClass({
     getInitialState: function() {
         return {
-            index: 0
+            index: 0,
+            sigs: []
```

Comment:
In any case, I think it is valuable, especially for new members, to be able to see a full list of SIGs and committees on the display, even if they don't meet regularly. Let's add them to the panel anyway without meeting times. If we add an optional meeting times field to the groups service, we can represent a null or undefined value as "None".

+1 for events teams -- I definitely want to see RP and HackIllinois on the display.